src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -2,156 +2,126 @@
 out: Basic-Def.html
 ---
 
-  [More-About-Settings]: More-About-Settings.html
+  [Task-Graph]: Task-Graph.html
   [Bare-Def]: Bare-Def.html
   [Full-Def]: Full-Def.html
   [Running]: Running.html
   [Library-Dependencies]: Library-Dependencies.html
   [Input-Tasks]: ../docs/Input-Tasks.html
 
-.sbt build definition
----------------------
+Build definition
+----------------
 
 This page describes sbt build definitions, including some "theory" and
-the syntax of `build.sbt`. It assumes you know how to [use sbt][Running]
+the syntax of `build.sbt`.
+It assumes you have installed a recent version of sbt, such as sbt 0.13.13,
+know how to [use sbt][Running],
 and have read the previous pages in the Getting Started Guide.
 
-### Three Flavors of Build Definition
+This page discusses the `build.sbt` build definition.
 
-There are three flavors of build definition:
+### Setting the sbt version
 
-1. Multi-project `.sbt` build definition
-2. Bare `.sbt` build definition
-3. `.scala` build definition
+First, create a file named `project/build.properties` to set
+a particular version. In this file, write:
 
-This page discusses the newest multi-project `.sbt` build definition, which combines the strength
-of the two older flavors, and is suitable for all cases.
-You might come across the other older flavors when dealing with builds in the wild.
-See [bare .sbt build definition][Bare-Def] and [.scala build definition][Full-Def] (later in Getting Started) for more
-on other flavors.
+```
+sbt.version=$app_version$
+```
 
-In addition, a build definition can contain files ending in `.scala`, located in the
-`project/` subdirectory of the base directory to define commonly used functions and values.
+to use sbt $app_version$.
+Without this file sbt will chose the sbt version installed on the machine,
+but setting the sbt version in
+`project/build.properties` avoids any potential confusion.
 
```

Comment:
The transition from the follow-along style of the previous chapter is slightly jarring here. Maybe something like this?

> ### Specifying the required sbt version
> As part of your build definition you will specify the version of sbt that your project requires. This allows people with different versions of the `sbt` launcher to build the same project with consistent results. To do this, your project should have a file named `project/build.properties` that specifies the required version as follows:
> ```
> sbt.version=$app_version$
> ```
> If the required version is not available locally, `sbt` launcher will download it for you.  If this file is not present, the `sbt` launcher will choose an arbitrary version, which is discouraged because it makes your project non-portable.